Mage.Sets/src/mage/cards/c/CallousOppressor.java

```diff
@@ -0,0 +1,160 @@
+ *  CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ *  SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON
+ *  ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+ *  NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF
+ *  ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ *  The views and conclusions contained in the software and documentation are those of the
+ *  authors and should not be interpreted as representing official policies, either expressed
+ *  or implied, of BetaSteward_at_googlemail.com.
+ */
+package mage.cards.c;
+
+import java.util.LinkedHashSet;
+import java.util.UUID;
+import java.util.stream.Collectors;
+import mage.MageInt;
+import mage.MageObject;
+import mage.abilities.Ability;
+import mage.abilities.common.AsEntersBattlefieldAbility;
+import mage.abilities.common.SimpleActivatedAbility;
+import mage.abilities.common.SkipUntapOptionalAbility;
+import mage.abilities.condition.common.SourceTappedCondition;
+import mage.abilities.costs.common.TapSourceCost;
+import mage.abilities.decorator.ConditionalContinuousEffect;
+import mage.abilities.effects.OneShotEffect;
+import mage.abilities.effects.common.continuous.GainControlTargetEffect;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.choices.Choice;
+import mage.choices.ChoiceImpl;
+import mage.constants.CardType;
+import mage.constants.SubType;
+import mage.constants.Duration;
+import mage.constants.Outcome;
+import mage.constants.Zone;
+import mage.filter.common.FilterCreaturePermanent;
+import mage.filter.predicate.Predicates;
+import mage.filter.predicate.mageobject.ChosenSubtypePredicate;
+import mage.game.Game;
+import mage.game.permanent.Permanent;
+import mage.players.Player;
+import mage.target.common.TargetCreaturePermanent;
+import mage.target.common.TargetOpponent;
+import mage.util.CardUtil;
+
+/**
+ *
+ * @author L_J
+ */
+public class CallousOppressor extends CardImpl {
+
+    public CallousOppressor(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId,setInfo,new CardType[]{CardType.CREATURE},"{1}{U}{U}");
+        this.subtype.add(SubType.CEPHALID);
+        this.power = new MageInt(1);
+        this.toughness = new MageInt(2);
+
+        // You may choose not to untap Callous Oppressor during your untap step.
+        this.addAbility(new SkipUntapOptionalAbility());
+
+        // As Callous Oppressor enters the battlefield, an opponent chooses a creature type.
+        this.addAbility(new AsEntersBattlefieldAbility(new OpponentChooseCreatureTypeEffect()));
+
+        // {T}: Gain control of target creature that isn't of the chosen type for as long as Callous Oppressor remains tapped.
+        FilterCreaturePermanent filter = new FilterCreaturePermanent("creature that isn't of the chosen type");
+        filter.add(Predicates.not(new ChosenSubtypePredicate(this.getId())));
```

Comment:
This could raise problems if the permanent is copied e.g. by clone. 
Did you test it?

Reply:
I've imitated the implementation from Belbe's Portal here, but you're right - similarily to #4041 it still considers the chosen subtype of the original for the filter; however, the player correctly gets a new prompt for an opponent (and thus a new subtype choice) and the blue tooltip text shows the newer chosen subtype and not the original. Should I make a new class for the target then?